core/src/main/java/bisq/core/btc/TxFeeEstimationService.java

```diff
@@ -41,9 +41,24 @@
  */
 @Slf4j
 public class TxFeeEstimationService {
-    public static int TYPICAL_TX_WITH_1_INPUT_SIZE = 260;
-    private static int DEPOSIT_TX_SIZE = 320;
-    private static int PAYOUT_TX_SIZE = 380;
+
+//  Size/vsize of typical trade txs
+//  Real txs size/vsize may vary in 1 or 2 bytes from the estimated values.
+//  Values calculated with https://gist.github.com/oscarguindzberg/3d1349cb65d9fd9af9de0feaa3fd27ac
+//  legacy fee tx with 1 input, maker/taker fee paid in btc size/vsize = 258
+//  legacy deposit tx without change size/vsize = 381
+//  legacy deposit tx with change size/vsize = 414
+//  legacy payout tx size/vsize = 337
+//  legacy delayed payout tx size/vsize = 302
+//  segwit fee tx with 1 input, maker/taker fee paid in btc vsize = 173
+//  segwit deposit tx without change vsize = 232
+//  segwit deposit tx with change vsize = 263
+//  segwit payout tx vsize = 169
+//  segwit delayed payout tx vsize = 139
+    public static int TYPICAL_TX_WITH_1_INPUT_SIZE = 175;
+    private static int DEPOSIT_TX_SIZE = 233;
+    private static int PAYOUT_TX_SIZE = 169;
```

Comment:
PAYOUT_TX_SIZE is not used anymore and can be deleted.

Reply:
Done